table_transforms/linear_reg/linear_reg.yaml

```diff
@@ -0,0 +1,28 @@
+name: linear_reg
+description: |
+  Fit a simple linear regression and return the formula. Optionally, use one or more group_by columns to create a regression per unique grouping.
+  
+  Currently, only supports a single independent variable.
+arguments:
+  group_by:
+    type: column_list
+    description: Optional, columns to group by before building the linear regression model. Use this field to create multiple models (one per unique grouping)
```

Comment:
Are you setting an optional variable the right way here?

Reply:
good call thanks